resources/assets/js/translations/extractedMessages/resources/assets/js/components/ApplicantSkills/SkillsWordCounterMessages.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
This file should be deleted, shouldn't it?

Reply:
Ya, that's wierd.